server/src/main/java/com/github/animelist/animelist/model/profilepage/StatisticsBlockAdditionalData.java

```diff
@@ -0,0 +1,52 @@
+package com.github.animelist.animelist.model.profilepage;
+
+public class StatisticsBlockAdditionalData {
+    private int entries;
+    private float avgRating;
```

Comment:
We can use `double` instead since graphql `Float` is actually double precision to be consistent